### PR TITLE
Labeled partner packs + cleared packs metadata

### PR DESCRIPTION
**Pull request by IkaDemisto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23986

## Packs labeled as partner support:
- Accessdata
- Aella_StarLight
- AlphaSOC_Network_Behavior_Analytics
- AlphaSOC_Wisdom
- BitDam
- CloudShark
- CounterCraft
- DomainTools
- ExtraHop
- Flashpoint
- GoogleChronicleBackstory
- Indeni
- Intezer
- Luminate
- Pcysys
- PerceptionPoint
- PhishAI
- PolySwarm
- SCADAfence_CNM
- SecBI
- SecurityAdvisor
- Skyformation
- SlashNextPhishingIncidentResponse
- SlashNextPhishingIncidentResponse
- ThreatQ

## Description
Labeled partners packs in pack metadata and cleared unused keys.
Other packs that are listed in https://github.com/demisto/etc/issues/23986#issuecomment-624857091 don't really match existing packs in content.

### CODE REVIEW

**Comment by anara123** [comment]
@IkaDemisto why you haven't updated the pack version? Because right now we are overriding the versions?

**Comment by IkaDemisto** [comment]
> @IkaDemisto why you haven't updated the pack version? Because right now we are overriding the versions?

@anara123 correct

File: Packs/Accessdata/pack_metadata.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "Accessdata",
   "description": "Use the Accessdata integration to protect against and provide additional visibility into phishing and other malicious email attacks.",
-  "support": "xsoar",
+  "support": "partner",
   "currentVersion": "1.0.0",
-  "author": "Cortex XSOAR",
-  "url": "https://www.paloaltonetworks.com/cortex",
+  "author": "Accessdata",
+  "url": "",
```
Comment (anara123) [inline Packs/Accessdata/pack_metadata.json:7]:
```suggestion
  "url": "https://support.accessdata.com/hc/en-us",
```

File: Packs/Accessdata/pack_metadata.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
   "email": "",
```
Comment (anara123) [inline Packs/Accessdata/pack_metadata.json:8]:
```suggestion
  "email": "support@accessdata.com",
```

File: Packs/PhishAI/pack_metadata.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
   "name": "Phish.AI",
   "description": "Next-Generation Anti-Phishing Platform Powered by AI & Computer Vision",
-  "support": "xsoar",
+  "support": "partner",
   "currentVersion": "1.0.0",
-  "author": "Cortex XSOAR",
-  "url": "https://www.paloaltonetworks.com/cortex",
+  "author": "Phish.AI",
+  "url": "",
   "email": "",
```
Comment (anara123) [inline Packs/PhishAI/pack_metadata.json:8]:
```suggestion
  "email": "info@phish.ai",
```

File: Packs/BitDam/pack_metadata.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
   "name": "BitDam",
   "description": "BitDam secure email gateway protects from advanced content-borne threats with the most accurate prevention of known and unknown threats, at their source.",
-  "support": "xsoar",
+  "support": "partner",
   "currentVersion": "1.0.0",
-  "author": "Cortex XSOAR",
-  "url": "https://www.paloaltonetworks.com/cortex",
+  "author": "BitDam",
+  "url": "",
   "email": "",
```
Comment (anara123) [inline Packs/BitDam/pack_metadata.json:8]:
```suggestion
  "email": "info@bitdam.com",
```

File: Packs/CounterCraft/pack_metadata.json [hunk 5]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "CounterCraft Deception Director",
   "description": "CounterCraft Deception Solution detects advanced adversaries. Automate counterintelligence campaigns to discover targeted attacks with real-time active response.",
-  "support": "xsoar",
+  "support": "partner",
   "currentVersion": "1.0.0",
-  "author": "Cortex XSOAR",
+  "author": "Counter Craft",
   "url": "https://www.paloaltonetworks.com/cortex",
```
Comment (anara123) [inline Packs/CounterCraft/pack_metadata.json:7]:
```suggestion
  "url": "",
```

**Pull request merged**